packages/editor-ui/src/components/MainHeader/CollaborationPane.vue

```diff
@@ -0,0 +1,73 @@
+<script setup lang="ts">
+import { COLLABORATION_HEARTBEAT_INTERVAL } from '@/constants';
+import { useUsersStore } from '@/stores/users.store';
+import { useWorkflowsStore } from '@/stores/workflows.store';
+import { useCollaborationStore } from '@/stores/collaboration.store';
+import { onBeforeUnmount } from 'vue';
+import { onMounted } from 'vue';
+import { computed, ref } from 'vue';
+
+const collaborationStore = useCollaborationStore();
+const usersStore = useUsersStore();
+const workflowsStore = useWorkflowsStore();
+
+const heartbeatInterval = COLLABORATION_HEARTBEAT_INTERVAL;
```

Comment:
Also somewhat a matter of personal pref, but since this constant is only used here, why define it in a landfill of multiple constants, when we could define it here?